01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
+
+ReminderZero = 0
+
+def fizzbuzz(start_num = 1 ,end_num = 20,fizz_num = 3,buzz_num = 5)
```

Comment:
```suggestion
def fizzbuzz(start_num = 1, end_num = 20, fizz_num = 3, buzz_num = 5)
```

こういう感じでスペースを使いますね。

あと、この設計だと「start_numだけ変えたい」というときは便利ですが、「buzz_numだけ変えたい」と思ったときは最初の3つも必ず指定しなきゃいけないのでちょっと不便です。
なので改善をお願いします・・・と言いたいところですが、FizzBuzz問題はそんなにプログラムの柔軟性を意識したりしなくて良いので引数は全部なくしてしまってよいです。